app/src/main/java/live/hms/app2/ui/meeting/MeetingViewModel.kt

```diff
@@ -86,7 +86,7 @@ class MeetingViewModel(
       synchronized(_tracks) {
         field = value
 
-        val volume = if (isAudioMuted) 0.0 else 1.0
+        val volume = if (isAudioMuted) 0.0 else 10.0
```

Comment:
Did you mean to turn the volume up to 10 here? @PratimMallick

Reply:
Wanted to check what is the effect if volume is set to greater than 1. It does seem to increase the gain rather than the volume. I will keep it at 1 only